src/c++/perf_analyzer/concurrency_manager.cc

```diff
@@ -100,6 +111,11 @@ ConcurrencyManager::ChangeConcurrencyLevel(
       }
     }
   }
+}
+
+void
+ConcurrencyManager::ReconfigThreads(const size_t concurrent_request_count)
```

Comment:
Is it true that concurrency only goes up between calls to this (and parent) function?

Reply:
This is unchanged cut and pasted code. 
I don't see anything yet that sorts the rates, but I haven't looked too closely. That would be for another story to investigate.